docs/preview/02-Features/powershell/azure-devops.md

```diff
@@ -126,7 +172,7 @@ PS> Save-AzDevOpsBuild -ProjectId $(System.TeamProjectId) -BuildId $(Build.Build
 # Information on them can be found here: https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml
 ```
 
-This function is intended to be used from an Azure DevOps pipeline.  Internally, it uses some predefined Azure DevOps variables.
+This function is intended to be used from an Azure DevOps pipeline. Internally, it uses some predefined Azure DevOps variables.
```

Comment:
Well, we can argue about that ..... :)

Seems like [only people over 40 use 2 spaces after a period](https://www.cultofpedagogy.com/two-spaces-after-period/#:~:text=Unless%20you%20are%20typing%20on,applies%20to%20all%20end%20punctuation.) ....   (I'm guilty to this as well).

I can agree that we use one space after a period, but then we need to be consistent, and do it everywhere :D

Reply:
Today I learned something I never expected :D I just noticed in this specific file the two spaces, so corrected it without really thinking about it. I just checked, and on 2 more places it is used. Should I also update it there?

Reply:
After my latest commit now all double spaces after a period are removed from this specific file :)

Reply:
Thanks